R/methods-XCMSnExp.R

```diff
@@ -2404,9 +2404,9 @@ setMethod("fillChromPeaks",
                       "OnDiskMSnExp",
                       processingData = new("MSnProcess",
                                            files = fileNames(object)[i]),
-                      featureData = new("AnnotatedDataFrame", fd),
-                      phenoData = new("NAnnotatedDataFrame",
-                                      data.frame(sampleNames = "1")),
+                      featureData = AnnotatedDataFrame(fd),
+                      phenoData = AnnotatedDataFrame(
+                          data.frame(sampleNames = "1")),
                       experimentData = new("MIAPE",
                                            instrumentManufacturer = "a",
```

Comment:
"1" and "a" are dummy values. Should one make that more visible ? "Dummy" ?

Reply:
AFAIK this is just some internal code to more efficiently process the data - the object with these values is never exported to the user.